WordPressAuthenticator.podspec

```diff
@@ -42,6 +41,6 @@ Pod::Spec.new do |s|
   # Use a loose restriction that allows both production and beta versions, up to the next major version.
   # If you want to update which of these is used, specify it in the host app.
   s.dependency 'WordPressUI', '~> 1.7-beta'
-  s.dependency 'WordPressKit', '~> 4.18-beta' # Don't change this until we hit 5.0 in WPKit
-  s.dependency 'WordPressShared', '~> 1.12-beta' # Don't change this until we hit 2.0 in WPShared
+  s.dependency 'WordPressKit', '~> 5.0.0-beta.1'
+  s.dependency 'WordPressShared', '~> 2.0.0-beta.1'
```

Comment:
For a while, I though we had to either use `~> 2.0.0-beta.1` and be constrained to `2.0.x` versions, or `'~> 2.0', '>= 2.0.0-beta.1'` and pay the price of a more verbose specification to get all the `2.x` version.

I just learned that we can use `~> 2.0-beta` to get the best of both world. Incidentally, that's also the style used in the previous version of this code.